Tests/UnitTests/Scripts/tests.js

```diff
@@ -20,12 +20,22 @@ mocha.setup({ ui: "bdd", reporter: BabylonReporter });
 const expect = chai.expect;
 
 describe("XMLHTTPRequest", function () {
-    function createRequest(method, url) {
+    function createRequest(method, url, body) {
         return new Promise((resolve) => {
             const xhr = new XMLHttpRequest();
             xhr.open(method, url);
             xhr.addEventListener("loadend", () => resolve(xhr));
-            xhr.send();
+            !body ? xhr.send() : xhr.send(body);
```

Comment:
Should be equivalent to this. JavaScript shouldn't know the difference.
```suggestion
            xhr.send(body);
```